gems/spree/OSVDB-91217.yml

```diff
@@ -0,0 +1,17 @@
+---
+gem: spree
+cve: 2013-1656
+osvdb: 91217
+url: https://spreecommerce.com/blog/multiple-security-vulnerabilities-fixed
+title: |
+  Spree payment_methods_controller.rb payment_method Parameter Arbitrary Ruby
+  Object Instantiation Command Execution
+date: 2013-02-21
+description: |
+  Spree contains a flaw that is triggered when handling input passed via the
+  'payment_method' parameter to payment_methods_controller.rb. This may allow
+  a remote authenticated attacker to instantiate arbitrary Ruby objects and
+  potentially execute arbitrary commands.
+cvss_v2: 4.3
+patched_versions:
+  - ">= 2.0.0"
```

Comment:
Looks like [spree 1.3.3](https://rubygems.org/gems/spree/versions) was released June 13, 2013. So probably should be `">= 1.3.3"`.